src/print.rs

```diff
@@ -403,3 +395,66 @@ pub fn format_duration(duration: &Duration) -> String {
         format!("{:?}ms", &milis)
     }
 }
+
+// Return the modules from $all that are not already in the list
+fn all_modules_uniq(module_list: &BTreeSet<String>) -> Vec<String> {
+    let mut prompt_order: Vec<String> = Vec::new();
+    for module in PROMPT_ORDER.iter() {
+        if !module_list.contains(*module) {
+            prompt_order.push(String::from(*module))
+        }
+    }
+
+    prompt_order
+}
+
+fn load_formatter_and_modules<'a>(context: &'a Context) -> (StringFormatter<'a>, BTreeSet<String>) {
```

Comment:
Love that this is broken out into a separate function! 👍 
Could you please add a descriptive comment for this function?